src/dispatcher.ts

```diff
@@ -47,6 +52,7 @@ export default class CommandDispatcher extends EventEmitter {
                 console.log('message intent: ' + data.entities.intent[0].value);
             }
         } else if (message.embeds.length > 0 || message.attachments.array().length > 0) {
+            this.checkImageForUsers(message);
```

Comment:
This function should be async and not stall, we need to avoid doing work in the dispatcher's callback method. I'll check it off for now, but this will cause delays between processing. I'll rework the dispatcher to better distribute work.